### PR TITLE
♿ [a11y-fixit][amp-accordion] Add various a11y attributes to amp-accordion

### PR DESCRIPTION
**Pull request by krdwan**
Add the `role` and `aria-labelledby` attributes to the `content` element of each `accordion` section.

Fixes: https://github.com/ampproject/amphtml/issues/30727

### CODE REVIEW

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 1]
```diff
@@ -170,6 +176,8 @@ class AmpAccordion extends AMP.BaseElement {
         header.setAttribute('tabindex', 0);
       }
       this.headers_.push(header);
+      content.setAttribute('aria-labelledby', headerId);
```
Comment (kristoferbaxter) [inline extensions/amp-accordion/0.1/amp-accordion.js:179]: Do these values need to be set after the `push` above or should they be set beforehand?

**Review by kristoferbaxter — APPROVED** [review]
Generally looks good, left a few comments.

File: extensions/amp-accordion/0.1/amp-accordion.js (hunk 1 above)
Comment (krdwan) [reply extensions/amp-accordion/0.1/amp-accordion.js:179]: They can be set before or after the `push`.  I just figured I'd keep all the `header` related lines together than do the `content` after.

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 2]
```diff
@@ -114,17 +114,24 @@ class AmpAccordion extends AMP.BaseElement {
...
       let contentId = content.getAttribute('id');
       if (!contentId) {
-        // To ensure that we pass Accessibility audits -
-        // we need to make sure that each accordion has a unique ID.
-        // In case the accordion doesn't have an ID we use a
-        // random number to ensure uniqueness.
         contentId = this.prefix_ + '_AMP_content_' + index;
         content.setAttribute('id', contentId);
       }
+      let headerId = header.getAttribute('id');
```
[the remark this answers is not in the document]
Comment (krdwan) [reply extensions/amp-accordion/0.1/amp-accordion.js:130]: Woohoo, I agree!

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 3]
```diff
@@ -114,17 +114,24 @@ class AmpAccordion extends AMP.BaseElement {
           'amp-accordion/amp-accordion.md. Found in: %s',
         this.element
       );
+      const header = sectionComponents[0];
```
[the remark this answers is not in the document]
Comment (krdwan) [reply extensions/amp-accordion/0.1/amp-accordion.js]: They do not, updated in the latest revision.

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 4]
```diff
@@ -114,17 +114,24 @@ class AmpAccordion extends AMP.BaseElement {
...
       const content = sectionComponents[1];
       content.classList.add('i-amphtml-accordion-content');
+
+      // To ensure that we pass Accessibility audits -
```
[the remark this answers is not in the document]
Comment (krdwan) [reply extensions/amp-accordion/0.1/amp-accordion.js]: Thanks, updated the comment!

**Pull request merged**